Code/Framework/AzCore/AzCore/IO/Streamer/FileRequest.h

```diff
@@ -21,403 +21,417 @@
 #include <AzCore/std/smart_ptr/shared_ptr.h>
 #include <AzCore/std/string/string_view.h>
 
-namespace AZ
+namespace AZ::IO
 {
-    namespace IO
-    {
-        class StreamStackEntry;
-        class ExternalFileRequest;
+    class StreamStackEntry;
+    class ExternalFileRequest;
 
-        using FileRequestPtr = AZStd::intrusive_ptr<ExternalFileRequest>;
-        
-        class FileRequest final
+    using FileRequestPtr = AZStd::intrusive_ptr<ExternalFileRequest>;
+    namespace Requests
```

Comment:
Alternatively for nesting inner namespace blocks is to open a namespace block just for the forward declarations and type aliases and then use the nested namespace feature to open the inner namespace blocks with the same indentation.

```
namespace AZ::IO
{
    class StreamStackEntry;
    class ExternalFileRequest;

    using FileRequestPtr = AZStd::intrusive_ptr<ExternalFileRequest>;
}

namespace AZ::IO::Requests
{
    // ...
}